### ISSUE TITLE
Color/texture no longer appears in facet filters

### ISSUE THREAD

**Issue by dgtn-va** [PrestaShop/PrestaShop#10869]
Hi,

I had to reset the module (Faceted Search) to re-index all products prices (since it doesn't work by itself ...), but suddenly my textures thumbnails missing and I've checkboxes displayed instead: ![tissus](https://user-images.githubusercontent.com/43678844/46521879-4217ea00-c881-11e8-851f-b03df4db0340.jpg)

And impossible (of course), to access the module config to check this point: ![faceted_search_module](https://user-images.githubusercontent.com/43678844/46521902-53f98d00-c881-11e8-98c8-e3706c4f94b0.jpg)

My attributes are correctly filled (since they worked well until then): 
![attributes](https://user-images.githubusercontent.com/43678844/46521872-3b897280-c881-11e8-9ac0-27ca1f8ee234.jpg)

I can't find anything in the DB to modify the potential parameter that would be appropriate.

PrestaShop version: 1.7.4.2
PHP version: 7.0

**Comment by khouloudbelguith** [comment]
Hi @fujitsuDev,

Thanks for your report.
Could you please try to re-install the Faceted Search module.

PS: Could you please try to configure your smarty by disabling the cache, set the option "Template compilation" to Force compilation and clear cache.
Thanks to check and feedback.

**Comment by dgtn-va** [comment]
Thanks for response.

Cache is already completly disabled and cleared many times. Module already re-installed too, two times.
Sorry, I should have said.
![performances](https://user-images.githubusercontent.com/43678844/46605408-ad6bf100-caf9-11e8-94c4-8c5d978b5fd9.jpg)

**Comment by khouloudbelguith** [comment]
Hi @fujitsuDev,

I did not manage to reproduce the issue with PS 1.7.4.3 & the ps_facetedsearch module v2.1.2.
![image](https://user-images.githubusercontent.com/16067358/46606140-1ce2e000-cafc-11e8-9f98-3f47731a150c.png)
What is the exact version of the ps_facetedsearch did you use?
Thanks!

**Comment by dgtn-va** [comment]
Prestashop v1.7.4.2
ps_facetedsearch v2.2.0, re-installed from https://github.com/PrestaShop/ps_facetedsearch/releases

**Comment by khouloudbelguith** [comment]
@fujitsuDev, could you please provide us with more info? We need more details to understand how we can reproduce your issue:
 - host
 - server setup and configuration
 - PrestaShop version (source)
 - debug mode report
 - PHP error logs
 - apache error log
 - javascript console log
 - screenshots

Don't you know how to get this information? Please read the following article:
http://build.prestashop.com/howtos/misc/how-to-create-bug-report/

Thanks!

**Comment by dgtn-va** [comment]
- Prestashop v1.7.4.2 upgraded one time from previous minor version with official autoupgrade module (https://github.com/PrestaShop/autoupgrade/releases)
- PHP 7.0, in 7.1 nothing works... php.ini file attached and http://php7-0.yulpa.io/
- No JS or Apache error in category products list page.
- Debug exception log of category products list page is attached.
- Theme ID: 32675 (child-theme applied)

It perfectly worked just before reitialize module, I didn't do anything else.

[ps_debug_exception.txt](https://github.com/PrestaShop/PrestaShop/files/2459853/ps_debug_exception.txt)
[php.ini.txt](https://github.com/PrestaShop/PrestaShop/files/2459864/php.ini.txt)

**Comment by khouloudbelguith** [comment]
@fujitsuDev, Thanks to navigate to your BO => IMPROVE => Design => Image Settings you can configure the IMAGE SETTINGS.
After that click on the button Regenerate thumbnails.
Thanks to check & feedback.

**Comment by dgtn-va** [comment]
It was useless, no texture thumbnail seems to have been regenerated...

**Comment by khouloudbelguith** [comment]
@fujitsuDev, Have you tried to contact the theme author?
Thanks to check the issue with them & feedback.

**Comment by dgtn-va** [comment]
The theme's author can't help. I checked: issue comes from PrestaShop, since there is always the problem on the "classic" theme.
Upgrade 1.7.4.2 to 1.7.4.3 has solved faceted search GUI issue from back-office, but nothing more.
I give up, I've too many bugs to try to fixe and I had to make choices... Sincerly, I've never "play" with such a buggy CMS.

Thanks again to have tried to solve the problem, sorry for the time lost.
